src/components/charts/asteroidClass/asteroid-class.module.scss

```diff
@@ -13,19 +13,46 @@
   }
 }
 
+.axis {
+  :global {
+    .tick {
+      @include labelPrimary;
+    }
+  };
+}
+
 .x-axis-label {
   text-anchor: middle;
 }
 
 .point {
   opacity: 0.2;
-  fill: $mediumBlue;
+  fill: $chart3;
```

Comment:
More like a heads up, I removed $chart4 from the colors as I was not able to find and acceptable 'yellow' that was passing the contrast test.